lib/wasi/src/syscalls/wasi/path_filestat_get.rs

```diff
@@ -92,3 +81,61 @@ pub(crate) fn path_filestat_get_internal(
     stat.st_ino = st_ino;
     Ok(stat)
 }
+
+/// ### `path_filestat_get_old()`
+/// Access metadata about a file or directory
+/// Inputs:
+/// - `Fd fd`
+///     The directory that `path` is relative to
+/// - `LookupFlags flags`
+///     Flags to control how `path` is understood
+/// - `const char *path`
+///     String containing the file path
+/// - `u32 path_len`
+///     The length of the `path` string
+/// Output:
+/// - `__wasi_file_stat_t *buf`
+///     The location where the metadata will be stored
+#[instrument(level = "trace", skip_all, fields(%fd, path = field::Empty), ret)]
+pub fn path_filestat_get_old<M: MemorySize>(
+    ctx: FunctionEnvMut<'_, WasiEnv>,
```

Comment:
As above

Reply:
I think it's safer to left them in the same file actualy